app/views/layouts/application.html.erb

```diff
@@ -10,39 +10,47 @@
   </head>
 
   <body>
-    <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #e3f2fd;"">
-      <% if current_user %>
-        <a class="navbar-brand" href="#">Main Menu</a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <!-- does this ^ do anything? -->
+    <div class="content">
+      <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #e3f2fd;"">
+        <% if current_user %>
+          <a class="navbar-brand" href="#">Main Menu</a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <!-- does this ^ do anything? -->
 
-        <div class="collapse navbar-collapse" id="navbarSupportedContent">
-          <ul class="navbar-nav mr-auto">
-            <li class="nav-item">
-              <a class="nav-link" <%= link_to "Dashboard", dashboard_path  %></a>
-            </li>
-            <li class="nav-item dropdown">
-              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Symptom Logs
-              </a>
-              <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                <a class="dropdown-item" href="#">View All Symptom Logs</a>
-                <div class="dropdown-divider"></div>
-                <a class="dropdown-item" href="#">Something else here</a>
-              </div>
-            </li>
-          </ul>
-            <span class="navbar-text">
-              <a class="nav-link" <%= link_to 'Sign Out', logout_path %></a>
-            </span>
-        </div>
-      <% end %>
-    </nav>
-    <div class="container">
-      <%= yield %>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item">
+                <a class="nav-link" <%= link_to "Dashboard", dashboard_path  %></a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  Symptom Logs
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                  <a class="dropdown-item" href="#">View All Symptom Logs</a>
+                  <div class="dropdown-divider"></div>
+                  <a class="dropdown-item" href="#">Something else here</a>
```

Comment:
we should finalize the nav bar and get rid of the placeholders if they're not being used

Reply:
Agreed.  I did wonder if maybe some things couldn't be moved to the navbar (ex: Edit Medication List, Add New Medication) to clean up the dashboard a bit, but that's not really necessary

Reply:
I feel like putting 'edit medication list' and 'add new medication' in the navbar would feel really strange. They're actions directly tied to a user's profile/information not general website navigation...

Reply:
^ agreed @gabichuelas